util/parseOptions.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Parse options string into object.
+ *
+ * @param {string} optionsString
+ * @returns {{}}
+ */
+module.exports = (optionsString) => {
+  let options = {};
+
+  optionsString.split("_").forEach((option) => {
+    const [action, ...params] = option.split(":");
+    // Currently only one parameter is supported.
+    options[action] = params[0];
```

Comment:
You can unpack the first entry automatically, like this:

```suggestion
    const [action, param] = option.split(":");
    options[action] = param;
```

Any parameters that are not specified in the destructuring syntax won't be assigned.
Example:

```js
const [foo, bar] = "Foo bar baz".split(" "); // "baz" is simply ignored and not assigned.
```